test/api/v3/integration/tasks/POST-create_task.test.js

```diff
@@ -0,0 +1,67 @@
+import {
```

Comment:
I think the structure for naming should actually be:

```
test/api/v3/integration/<route_namespace>/<REST_ACTION>-<route_path>.test.js
```

So this one would be

```
test/api/v3/integration/tasks/POST-tasks.test.js
```

The idea is to make the integration tasks as agnostic about the actual workings of the controllers as possible. All they know is that they give a route some data, and get some data back.